src/app/garfile/responsibleperson/validations.js

```diff
@@ -59,12 +61,14 @@ module.exports.validations = (req) => {
       new ValidationRule(validator.isAddressValidCharacters, 'responsibleTown', responsibleTown, `Town or city must not contain special characters such as $ % ' or ä`),
     ],
     [
-      new ValidationRule(validator.notEmpty, 'responsiblePostcode', responsiblePostcode, 'Enter a postcode for the responsible person.'),
```

Comment:
Question: Why was this validator removed?

Reply:
Careless mistake by me.  I deleted it by accident.